Description: Update autoconf files for pjproject
 config.guess and config.sub for pjproject are six years old, this
 makes the build FTBFS on newer architectures like ppc64el.
 .
 Unfortunately the sources are only unpacked during the toplevel
 ./configure run, so we cannot solve this with dh_autoreconf
Author: Bernhard Schmidt <berni@debian.org>
Last-Update: 2018-12-21
--- a/third-party/pjproject/Makefile
+++ b/third-party/pjproject/Makefile
@@ -102,6 +102,11 @@ source/.unpacked: $(DOWNLOAD_DIR)/$(TARB
 		mv pjproject-$(PJPROJECT_VERSION) source
 	$(ECHO_PREFIX) Applying patches "$(realpath patches)" "$(realpath .)/source"
 	$(CMD_PREFIX) ../apply_patches $(QUIET_CONFIGURE) "$(realpath patches)" "$(realpath .)/source"
+	# Debian patch: Update autoconf files to support newer architectures
+	cd source && autoconf aconfigure.ac >aconfigure
+	cp /usr/share/misc/config.guess source/config.guess
+	cp /usr/share/misc/config.sub source/config.sub
+	# End Debian patch
 	-@touch source/.unpacked
 
 source/user.mak: $(if $(PJPROJECT_BUNDLED_OOT),,source/.unpacked) patches/user.mak
